src/app/DIAmeterApplication.cpp

```diff
@@ -0,0 +1,1241 @@
+                    if (nCandPeptide == 0) { continue; }
+
+                    TideMatchSet::Arr2 match_arr2(candidatePeptideStatusSize); // Scored peptides will go here.
+                    // Programs for taking the dot-product with the observed spectrum are laid
+                    // out in memory managed by the active_peptide_queue, one program for each
+                    // candidate peptide. The programs will store the results directly into
+                    // match_arr. We now pass control to those programs.
+                    TideSearchApplication::collectScoresCompiled(active_peptide_queue, spectrum, observed, &match_arr2, candidatePeptideStatusSize, charge);
+
+                    // The denominator used in the Tailor score calibration method
+                    double quantile_score = getTailorQuantile(&match_arr2);
+                    // carp(CARP_DETAILED_DEBUG, "Tailor quantile_score:%f", quantile_score);
+
+                    TideMatchSet::Arr match_arr(nCandPeptide);
+                    for (TideMatchSet::Arr2::iterator it = match_arr2.begin(); it != match_arr2.end(); ++it) {
+                       /// The code below which is adopted from Tide-search
+                       int peptide_idx = candidatePeptideStatusSize - (it->second);
+                       if ((*candidatePeptideStatus)[peptide_idx]) {
+                          TideMatchSet::Scores curScore;
+                           curScore.xcorr_score = (double)(it->first / XCORR_SCALING);
+                           curScore.rank = it->second;
+                           curScore.tailor = ((double)(it->first / XCORR_SCALING) + 5.0) / quantile_score;
+                           match_arr.push_back(curScore);
+                           // carp(CARP_DETAILED_DEBUG, "peptide_idx:%d \t xcorr_score:%f \t rank:%d", peptide_idx, curScore.xcorr_score, curScore.rank);
+                       }
+                    }
+
+                    TideMatchSet matches(&match_arr, highest_ms2_mz);
+                    if (!match_arr.empty()) {
+                        reportDIA(output_file, origin_file, spec_charge_chunk.at(chunk_idx), active_peptide_queue, proteins, locations,
+                                &matches,
+                                &observed,
+                                &ms1scan_mz_intensity_rank_map,
+                                &ms1scan_slope_intercept_map,
+                                &ms2scan_mz_intensity_map,
+                                &peptide_predrt_map);
+                    }
+
+                    delete min_mass;
+                    delete max_mass;
+                    delete candidatePeptideStatus;
+                }
+
+                // clear up for next chunk
+                for (map<int, boost::tuple<double*, double*, int>>::const_iterator i = ms2scan_mz_intensity_map.begin(); i != ms2scan_mz_intensity_map.end(); i++) { delete[] (i->second).get<0>(); delete[] (i->second).get<1>(); }
```

Comment:
Since this for loop has 2 internal code lines. I would put a line break between the two lines.